src/main/resources/okta/js/applications.js

```diff
@@ -0,0 +1,618 @@
+;(function( $ ) {
+    $.fn.zclip = function() {
+        if(typeof ZeroClipboard == 'function'){
+            var client = new ZeroClipboard( this );
+            client.on( "ready", function( readyEvent ) {
+              client.on( "aftercopy", function( event ) {
+                var target = $(event.target);
+                target.attr("title","Copied!")
+                target.tooltip('enable');
+                target.tooltip("show");
+                target.tooltip('disable');
+              });
+            });
+        }else{
+            console.warn('Warning : Dependency missing - ZeroClipboard Library');
+        }
+        return this;
+    };
+}( jQuery ));
+
+
+var GrantTypes = function (available) {
```

Comment:
Shall we add comments for the newly added java script functions. And shall we change the name to be meaningful GetGrantTypes ?

Reply:
This from production code
https://github.com/wso2/carbon-apimgt/blob/v6.1.66/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/js/applications.js#L22